### ISSUE TITLE
Decrease the Python heap to 64ko

### ISSUE THREAD

**Issue by fime-space** [Omega-Numworks/Omega#540]
**Is your feature request related to a problem? Please describe.**
As it has [been said](https://github.com/Omega-Numworks/Omega/issues/357), increasing the python heap also mean reduce the heap for the rest.
I noticed that some external apps (hexedit and sometimes KihCas) make the calculator reseting;
thats maybe due to a memory conflict ?

**Describe the solution you'd like**
I think that would be safer to decrase the heap to 64ko; I think that's enough for a lot of big scripts and wouldn't affect them.

**Comment by RedGl0w** [comment]
> As it has [been said](https://github.com/Omega-Numworks/Omega/issues/357#issuecomment-648279105), increasing the python heap also mean reduce the heap for the rest.

This is true : when the appsContainerStorage gets bigger, the native heap becomes smaller (but it is not used frequently in epsilon, so it shouldn't cause issues).
> I noticed that some external apps (hexedit and sometimes KihCas) make the calculator reseting;

- For hexedit, I have never noticed crashes (except when going on unmapped memory, which is totally normal)
- For KhiCas, we haven't been able to find the origin of the crash yet, but when we changed python heap, we also increased the external heap.
> I think that would be safer to decrase the heap to 64ko; I think that's enough for a lot of big scripts and wouldn't affect them.

This idea would be nice, because it would allow to increase storage (I don't think 100Ko for python heap and 32Ko for storage is a good ratio...)
